package.json

```diff
@@ -15,26 +15,28 @@
     "@types/node": "^12.0.0",
     "@types/react": "^17.0.3",
     "@types/react-dom": "^17.0.0",
+    "@types/react-grid-layout": "^1.1.1",
     "lodash": "^4.17.21",
+    "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-grid-layout": "^1.2.4",
     "react-redux": "^7.2.3",
     "react-scripts": "4.0.3",
+    "redux": "^4.0.5",
     "redux-devtools-extension": "^2.13.9",
     "redux-thunk": "^2.3.0",
-    "typescript": "^4.1.2",
+    "typescript": "^4.2.3",
     "web-vitals": "^1.0.1"
   },
+  "prettier": {
+    "tabWidth": 2,
+    "useTabs": false,
+    "printWidth": 100
+  },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
+    "lint-with-error-on-warns": "node utilsScripts/prepushLint.js"
```

Comment:
Skrypt który można puścić żeby sprawdzić lintowanie, jeśli na przykłąd pipeline nie przechodzi i chcesz sprawdzić to lokalnie.
Wtedy potrzeba napisać:
`npm run lint-with-error-on-warns`